apps/studio/components/interfaces/Database/Roles/CreateRolePanel.tsx

```diff
@@ -91,7 +89,7 @@ const CreateRolePanel = ({ visible, onClose }: CreateRolePanelProps) => {
         validate={validate}
         onSubmit={onSubmit}
       >
-        {({ isSubmitting, handleReset, values, initialValues }: any) => {
+        {() => {
```

Comment:
Can this anon function be removed entirely and just pass the props directly?

Reply:
sorry this is what I meant: https://github.com/supabase/supabase/pull/19718/commits/7bad052efbdbd96d26607899ca8b04f506680cd2